odoo/addons/base/tests/test_expression.py

```diff
@@ -15,20 +15,55 @@
 from odoo import Command
 
 
-class TestExpression(SavepointCaseWithUserDemo):
+class TransactionExpressionCase(TransactionCase):
+
+    def _search(self, model, domain, init_domain=None, test_complement=True):
```

Comment:
I would define `TransactionDomainCase` here so that it can be imported in test_new_api. I would like to avoid moving all tests in this PR. From what I understand, some tests are here to be ran before any other module.